CODEOWNERS

```diff
@@ -0,0 +1,8 @@
+# Global code owners: co-chairs
```

Comment:
Can you add a comment at the top with something about how peer review and additional approvals still apply and  refers to CONTRIBUTING guide and governance?